### PR TITLE
Create `tomsfastmath` formula, and use it for `clamav`

### PR DESCRIPTION
**Pull request by triallax**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

See https://github.com/Homebrew/homebrew-core/pull/100780#issuecomment-1118289790.

The test I wrote fails on 12-arm64, but I can't reproduce locally. Help would be appreciated.

### CODE REVIEW

File: Formula/tomsfastmath.rb [hunk 1]
```diff
@@ -0,0 +1,64 @@
+class Tomsfastmath < Formula
+  desc "Fast large integer arithmetic library written in portable ISO C"
+  homepage "https://www.libtom.net/TomsFastMath/"
+  url "https://github.com/libtom/tomsfastmath/releases/download/v0.13.1/tfm-0.13.1.tar.xz"
+  sha256 "47c97a1ada3ccc9fcbd2a8a922d5859a84b4ba53778c84c1d509c1a955ac1738"
+
+  depends_on "libtool" => :build
+
+  # Fixes some issues in `makefile.shared`, remove on next release.
+  patch :DATA
```
Comment (mhmdanas) [inline Formula/tomsfastmath.rb:10]: The changes in this patch were already done upstream, but a release with the changes hasn't been made yet.

File: Formula/tomsfastmath.rb [hunk 2]
```diff
@@ -0,0 +1,64 @@
+class Tomsfastmath < Formula
+  desc "Fast large integer arithmetic library written in portable ISO C"
+  homepage "https://www.libtom.net/TomsFastMath/"
+  url "https://github.com/libtom/tomsfastmath/releases/download/v0.13.1/tfm-0.13.1.tar.xz"
+  sha256 "47c97a1ada3ccc9fcbd2a8a922d5859a84b4ba53778c84c1d509c1a955ac1738"
```
Comment (mhmdanas) [inline Formula/tomsfastmath.rb:5]: I'm aware that there should be a license here, but I'm not sure what to put. The current development branch is under the Unlicense, but the latest release (0.13.1) is dual-licensed under some custom public domain dedication and the WTFPL. See https://github.com/libtom/tomsfastmath/blob/v0.13.1/LICENSE.

What should I do?

**Comment by mhmdanas** [comment]
Bump, waiting for answers for my question above.

File: Formula/tomsfastmath.rb (hunk 2 above)
Comment (SMillerDev) [reply Formula/tomsfastmath.rb:5]: Could you ask for a release?

Comment (mhmdanas) [reply Formula/tomsfastmath.rb:5]: Done: https://github.com/libtom/tomsfastmath/issues/29

Comment (mhmdanas) [reply Formula/tomsfastmath.rb]: My issue hasn't received any response, so I think we can put the dual license of the 0.13.1 release here?

Comment (mhmdanas) [reply Formula/tomsfastmath.rb:5]: Done. Is `:public_domain` appropriate for a custom public domain dedication?

**Comment by mhmdanas** [comment]
Bump, see question above.

**Pull request closed without merging**